examples/local_hugging_face_server.py

```diff
@@ -0,0 +1,89 @@
+"""Example server definition for deploying Huggingface models to a local endpoint.
+Sets up the /completions endpoint which (broadly) conforms to what's described
+in the link below.
+https://alexatallah.notion.site/RFC-LLM-API-Standard-c8f15d24bd2f4ab98b656f08cdc1c4fb#e891724fc18f4524a970aca2379e8985
+Main thing to note is the `import_and_return` functions which download the
+Huggingface model and returns the necessary components to use it.
+This connector function is stored in the `CONNECTED_MODELS` dictionary.
+"""
+from flask import Flask, jsonify, request
+from transformers import AutoTokenizer, AutoModelForSeq2SeqLM
+from transformers import BertTokenizer, BertForSequenceClassification
+from transformers import pipeline
+
+app = Flask(__name__)
+
+
+"""Connector functions."""
+
+def import_and_return_tk_instruct_3b_def():
+    """Example import function for Huggingface model. Returns model, tokenizer,
+    and code for running the model."""
+    tokenizer = AutoTokenizer.from_pretrained("allenai/tk-instruct-3b-def")
```

Comment:
optional idea to expand this: wonder if we could make the model name a variable? e.g. use the `modelId` parameter in the request POST body and do a cached tokenizer/model load